### PR TITLE
feat: Add fonts

### PR DESCRIPTION
**Pull request by Soap-141**
GitHub Issue: #

## PR Type

What kind of change does this PR introduce?

- Feature

## Description

Added fonts.

Before:
image

After:
image

## PR Checklist 
Please check if your PR fulfills the following requirements:

- [ ] Interface members are XML documented
- [ ] Documentation (XML or comments) has been added and/or existing documentation has been updated
- [ ] [Architecture documents](./doc/Architecture.md) have been updated
- [ ] Tested on all relevant platforms

### CODE REVIEW

**Comment by takla21** [comment]
no screenshots?

File: src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj [hunk 1]
```diff
@@ -11,6 +11,16 @@
 		<SupportedOSPlatformVersion Condition="'$(TargetFramework)' == 'net6.0-android'">26.0</SupportedOSPlatformVersion>
 		<SupportedOSPlatformVersion Condition="'$(TargetFramework)' == 'net6.0-ios'">14.2</SupportedOSPlatformVersion>
 	</PropertyGroup>
+	<ItemGroup>
+	  <None Remove="Android\Assets\Fonts\Guillon-Black.ttf" />
```
Comment (takla21) [inline src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj:15]: Are they actually any style that uses those fonts?

Comment (Soap-141) [reply src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj:15]: Don't know, will have to check, I just added what was in the old template. I can also provide screenshots if it does. Also I see that the file as been updated, I will need to check that out.

Comment (Soap-141) [reply src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj]: Feel like it's working on Windows. I have to check mobile.

Before:
image

After:
image

**Review by takla21 — APPROVED** [review]

**Comment by vincentcastagna** [comment]
I wonder at which point we would like to bump to Uno.UI 7 to benefits from the fact that it is now unifying the way to include fonts for all heads. 
@jeanplevesque  what do you think ? I believe its a really nice [feature  ](https://platform.uno/docs/articles/features/custom-fonts.html)...

File: src/app/ApplicationTemplate.Windows/ApplicationTemplate.Windows.csproj [hunk 2]
```diff
@@ -19,6 +19,7 @@
 	</PropertyGroup>
 	
 	<ItemGroup>
+		<!-- TODO: Add missing images? -->
```
Comment (vincentcastagna) [inline src/app/ApplicationTemplate.Windows/ApplicationTemplate.Windows.csproj:22]: Which images are missing ?

**Review by vincentcastagna — APPROVED** [review]

File: src/app/ApplicationTemplate.Windows/ApplicationTemplate.Windows.csproj (hunk 2 above)
Comment (Soap-141) [reply src/app/ApplicationTemplate.Windows/ApplicationTemplate.Windows.csproj:22]: image

File: src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj [hunk 3]
```diff
@@ -201,6 +211,14 @@
 				<BundleResource Include="iOS/Resources/Assets/splashscreen_icon.png" />
 				<BundleResource Include="iOS/Resources/Assets/splashscreen_icon%402x.png" />
 				<BundleResource Include="iOS/Resources/Assets/splashscreen_icon%403x.png" />
+				<BundleResource Include="iOS\Resources\Fonts\Guillon-Black.ttf" />
```
Comment (carlh98) [inline src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj:214]: I'm curious the slash are not the same directions for the Assets and the Fonts, does that change something?

Comment (Soap-141) [reply src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj:214]: That's my bad, both are working, but it's a mix of VS adding the path by itself and me copying from other configuration file.

**Comment by jeanplevesque** [comment]
> I wonder at which point we would like to bump to Uno.UI 7 to benefits from the fact that it is now unifying the way to include fonts for all heads. @jeanplevesque what do you think ? I believe its a really nice [feature ](https://platform.uno/docs/articles/features/custom-fonts.html)...

Yeah that would be a nice thing to try.

**Comment by Soap-141** [comment]
> > I wonder at which point we would like to bump to Uno.UI 7 to benefits from the fact that it is now unifying the way to include fonts for all heads. @jeanplevesque what do you think ? I believe its a really nice [feature ](https://platform.uno/docs/articles/features/custom-fonts.html)...
> 
> Yeah that would be a nice thing to try.

I'll do it in a separate PR later today.

**Review by jeanplevesque — APPROVED** [review]

File: src/app/ApplicationTemplate.Windows/ApplicationTemplate.Windows.csproj (hunk 2 above)
Comment (Soap-141) [reply src/app/ApplicationTemplate.Windows/ApplicationTemplate.Windows.csproj:22]: Will be taking a look at this later for cleaning purposes.

File: src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj (hunk 3 above)
Comment (Soap-141) [reply src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj:214]: Will be cleaning this later today.

**Pull request merged**